<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stegano</class>
 <widget class="QWidget" name="Stegano">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stegano</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="KUrlComboRequester" name="cFile"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Verschl체sselung</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Passwort</string>
            </property>
            <property name="buddy">
             <cstring>tPassword</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KRestrictedLine" name="tPassword">
            <property name="showClearButton" stdset="0">
             <bool>true</bool>
            </property>
            <property name="passwordMode">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Hash</string>
            </property>
            <property name="buddy">
             <cstring>tHash</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="tHash">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="cUseEncryption">
            <property name="text">
             <string>Verschl체sseln</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Geheimnisse Verbergen..</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Vorschau</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="KImageFilePreview" name="preview"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="KPushButton" name="buttonUnhide">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Sichtbar machen</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="KPushButton" name="buttonSave">
                 <property name="text">
                  <string>Speichern</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Botschaft</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="font">
                    <font>
                     <pointsize>6</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>Zeichenvorrat:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="lCharsAvailable">
                   <property name="font">
                    <font>
                     <pointsize>6</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>0</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="KTabWidget" name="ktabwidget">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="tabText">
                  <attribute name="title">
                   <string>Text</string>
                  </attribute>
                  <layout class="QVBoxLayout" name="verticalLayout_9">
                   <item>
                    <widget class="KTextEdit" name="tMessageText"/>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="tabAttachment">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <attribute name="title">
                   <string>Dokumente</string>
                  </attribute>
                  <layout class="QVBoxLayout" name="verticalLayout_10">
                   <item>
                    <widget class="KListWidget" name="klistwidget"/>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="KPushButton" name="buttonAttach">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Anh채nge hinzuf체gen</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="KPushButton" name="buttonHide">
                 <property name="text">
                  <string>Verbergen</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KImageFilePreview</class>
   <extends>QWidget</extends>
   <header>kimagefilepreview.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlComboRequester</class>
   <extends>KUrlRequester</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KRestrictedLine</class>
   <extends>KLineEdit</extends>
   <header>krestrictedline.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cFile</tabstop>
  <tabstop>tPassword</tabstop>
  <tabstop>cUseEncryption</tabstop>
  <tabstop>tMessageText</tabstop>
  <tabstop>buttonHide</tabstop>
  <tabstop>buttonSave</tabstop>
  <tabstop>buttonAttach</tabstop>
  <tabstop>ktabwidget</tabstop>
  <tabstop>tHash</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cFile</sender>
   <signal>urlSelected(KUrl)</signal>
   <receiver>preview</receiver>
   <slot>showPreview(KUrl)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>passwordChanged(QString)</signal>
  <slot>calculateHash(QString)</slot>
  <slot>showPreview()</slot>
 </slots>
</ui>
